packages/core/src/events/drag-period.class.ts

```diff
@@ -0,0 +1,16 @@
+export class DragPeriod {
```

Comment:
Could this be generalized into something like `TimeRange` or `TimePeriod`

Reply:
sure!